_scripts/otel-collector.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -ex
+
+
+if [ "$TRACING_OTEL_COLLECTOR_ENABLED" == "true" ]
+then
+
+  EXPORTERS="logging"
+
+  if [ "$TRACING_OTEL_COLLECTOR_GCP_ENABLED" == "true" ]
+  then
+    EXPORTERS="$EXPORTERS,googlecloud"
+  fi
+
+  if [ "$TRACING_OTEL_COLLECTOR_JAEGER_ENABLED" == "true" ]
+  then
+    EXPORTERS="$EXPORTERS,jaeger"
+  fi
+
+  echo -e "Starting otel collector to capture client traces.\n $EXPORTERS"
+
+  echo -e "$HOME"
```

Comment:
Do you need this?

Reply:
Thanks, that was debugging cruft. Will remove.